manifests/01-role-binding.yaml

```diff
@@ -16,3 +16,23 @@ roleRef:
   kind: Role
   apiGroup: rbac.authorization.k8s.io
   name: ingress-operator
+---
+# RoleBinding for the operator to delete Role and RoleBindings
+# in the openshift-config namespace.
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ingress-operator
```

Comment:
is this an established naming scheme for rolebindings?

Reply:
I have seen a number of operators simply name their role/rolebinding after the opreator. In this case, the operators seems to follow [this convention](https://github.com/openshift/cluster-ingress-operator/blob/a5525f387106974730ccb9f422e05115498cbc47/manifests/01-role-binding.yaml#L5).